packages/azure_functions/data_stream/functionapplogs/elasticsearch/ingest_pipeline/default.yml

```
@@ -44,6 +44,36 @@ processors:
       field: json.operationName
       target_field: azure.operation_name
       ignore_missing: true
+  - set:
+      field: temp_properties
+      value: '{{json.properties}}'
+      if: "ctx.json?.properties instanceof String"
+      ignore_failure: true
+      description: 'Temporarily store the properties string.'
+  - script:
+      lang: painless
+      source: >
+        if (ctx?.json?.properties instanceof String) {
+          String fixed = ctx.json.properties.replace("'", "\"");
```

```suggestion
          ctx.temp_properties = ctx.json.properties.replace("'", "\"");
```
and remove the below line `ctx.temp_properties = fixed;`